### ISSUE TITLE
Internal Notes field needs to have `blank=True`

### ISSUE THREAD

**Issue by ssciolla** [tl-its-umich-edu/canvas-app-explorer#276]
Currently, after #264 was merged, the `internal_notes` is present and works, but if you try to edit a tool without entering anything in the admin UI field for it, a validation error is thrown. Admins should not be required to add content without meaning or whitespace to get the form to submit.

The solution should be add to `blank=True` to the model field for `internal_notes`. This will also require a new migration to be generated I believe.

See https://docs.djangoproject.com/en/4.1/ref/models/fields/#blank

**Comment by roujiawang** [comment]
I have created a [new pull request](https://github.com/tl-its-umich-edu/canvas-app-explorer/pull/278) that awaits review to resolve this issue. Local test on my computer indicates that a card can now be created with a blank internal note field.

**Comment by ssciolla** [comment]
Test plan:

1) Go to a tool in the admin without internal notes.
2) Make some change to another field, then try to save it.

The form should work, rather than throw a validation error that `internal_notes` is required.
